executor-scripts/linux/bridge

```diff
@@ -126,6 +132,9 @@ set_bridge_opts() {
 	[ -x /sbin/brctl ] && set_bridge_opts_brctl && return 0
 }
 
+port_is_vxlan_evpn() {
+	echo yesno "$(ifquery -p vxlan-evpn "${1}" 2>/dev/null || true)"
```

Comment:
What does this actually do?  You either get “yesno yes” or “yesno no” as output from running this.  Not 1 as used elsewhere in the script.

Reply:
Fixed in the new patch. I've added the `echo` for some test reasons and I forgot to remove it.